### PR TITLE
Documentation for previewing

### PR DESCRIPTION
**Pull request by CrossBound**
I've added some documentation for the previewing section. Please let me know if this is all correct or if there's any additional information that should be included.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/PiranhaCMS/piranha.core.docs?pullRequest=59) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/PiranhaCMS/piranha.core.docs?pullRequest=59) before we can accept your contribution.<br/><br/>:x: CrossBound [sign now](https://cla.dotnetfoundation.org/PiranhaCMS/piranha.core.docs?pullRequest=59)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/PiranhaCMS/piranha.core.docs?pullRequest=59) it.</sub>

File: src/03. Application/06. Middleware.md [hunk 1]
```diff
@@ -187,7 +189,7 @@ Adds the middleware that resolves requests to the site root. It listens to URL's
 
 `GET /`
 
-The request is rewritten the same way as for pages.
+The startpage middleware will attempt to find a page with the slug `/home` and if found the request will be handled the same as a normal page.
```
Comment (tidyui) [inline src/03. Application/06. Middleware.md:192]: Hi @CrossBound! This is not completely accurate. The start page middleware executes api.Pages.GetStartpage() which will load the page with ParentId == null && SortOrder == 0, i.e the first page on root level in the site. It has nothing to do with the slug of the page.

Regards

Comment (CrossBound) [reply src/03. Application/06. Middleware.md:192]: Interesting, in that case, the documentation here https://piranhacms.org/docs/application/routing might not be accurate. "Given that the StartpageMiddleware is registered the page with the slug /home will be resolved."

Comment (CrossBound) [reply src/03. Application/06. Middleware.md:192]: How about we just replace that line with

"The start page middleware executes `api.Pages.GetStartpage()` which will load the page with `ParentId == null && SortOrder == 0` (i.e the first page on root level in the site). The request will then be handled the same as any other page."

Comment (tidyui) [reply src/03. Application/06. Middleware.md]: @CrossBound I see you point with the current docs referencing `/home`. I think it was meant as an example as the first section stated that there were two pages added in the structure, one with the slug `/home` and one with the slug `/about-us`. But the update text is less confusing!

**Pull request merged**